CMakeLists.txt

```diff
@@ -14,40 +14,38 @@ endif()
 find_package(ament_cmake_ros REQUIRED)
 
 find_package(image_geometry REQUIRED)
-find_package(OpenCV REQUIRED)
+find_package(OpenCV REQUIRED COMPONENTS core)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(sensor_msgs REQUIRED)
 
 add_library(DepthImageToLaserScan
   src/DepthImageToLaserScan.cpp
 )
-ament_target_dependencies(DepthImageToLaserScan
-  "image_geometry"
-  "OpenCV"
-  "sensor_msgs"
-)
+target_link_libraries(DepthImageToLaserScan PUBLIC
+  image_geometry::image_geometry
+  opencv_core
+  ${sensor_msgs_TARGETS})
+
 generate_export_header(DepthImageToLaserScan EXPORT_FILE_NAME ${PROJECT_NAME}/DepthImageToLaserScan_export.h)
 target_include_directories(DepthImageToLaserScan PUBLIC
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>"
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-  "$<INSTALL_INTERFACE:include>"
+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>"
 )
 
 add_library(DepthImageToLaserScanROS
   src/DepthImageToLaserScanROS.cpp
 )
-ament_target_dependencies(DepthImageToLaserScanROS
-  "rclcpp"
-  "rclcpp_components"
-)
-target_link_libraries(DepthImageToLaserScanROS DepthImageToLaserScan)
+target_link_libraries(DepthImageToLaserScanROS PUBLIC
+  rclcpp::rclcpp)
+target_link_libraries(DepthImageToLaserScanROS PRIVATE
+  rcutils::rcutils
```

Comment:
I know this isn't of your doing, but `DepthImageToLaserScanROS.cpp` doesn't actually need to include `rcutils`, and hence doesn't need to directly link against it.  So I'd be in favor of removing that unnecessary include, and removing this.

(rcutils also isn't properly exposed as a dependency in package.xml or CMakeLists.txt)

Reply:
Removed traces of `rcutils` in 9926f09